env0/data_variable_set.go

```diff
@@ -0,0 +1,83 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataVariableSet() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataVariableSetRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "the name of the variable set",
+				Required:    true,
+			},
+			"scope": {
```

Comment:
I think calling this `creationScope` is clearer, will also match our API

Reply:
we could do "creation_scope".
However, in all other places in the provider, it's scope.
So I prefer to keep it aligned.

Reply:
Alright, I already approved regardless 😄